vignettes/web_only/repFilter_v3.Rmd

```diff
@@ -0,0 +1,326 @@
+
+```{r example-1.2}
+# method = "by.meta", query = list(Lane = exclude("A"))
+immdata$meta %>% filter(Status != 'C')
+names(immdata$data)
+```
+
+Filter for samples where *Lane* is *B* or *C*:
+
+```{r example-1.3}
+# method = "by.meta", query = list(Lane = include("B", "C"))
+immdata$meta %>% filter(Lane == 'C' | Lane == 'B')
+names(immdata$data)
+```
+
+Filter out samples where  *Lane* is not *A* and not *C*
+
+```{r example-1.4}
+# method = "by.meta", query = list(Lane = exclude("B", "C"))
+immdata$meta %>% filter(Lane != 'C' & Lane != 'A')
+names(immdata$data)
+```
+
+#### Example 2
+Use  `interval`, `lessthan`(from minus infinity to your value) or `morethan`(from your value to plus affinity) to define interval values in the filter statement. These options accept  float.
+
+Filter for samples where *Age* is lower than *23*:
+```{r example-2.1}
+#method = "by.meta", query = list(Age = lessthan(23)) 
+immdata$meta %>% filter(Age < 23)
+names(immdata$data)
+```
+
+Filter for samples where *Age* is upper than *15*:
+
+```{r example-2.2}
+# method = "by.meta", query = list(Age = morethan(15))
+immdata$meta %>% filter(Age > 15)
+names(immdata$data)
+```
+
+Filter for samples where *Age* is between *15* and *23*:
+
+```{r example-2.3}
+# method = "by.meta", query = list(Age = interval(15, 23))
+immdata$meta %>% filter(Age > 15 & Age < 23)
+names(immdata$data)
+```
+
+You can also use **multiple conditions**
+
+Filter for samples where *Age* is between *15* and *23* and *Lane* is *B*:
+
+```{r example-2.4}
+# method = "by.meta", query = list(Age = interval(15, 23), Lane = include('B'))
+immdata$meta %>% filter(Age > 15 & Age < 23 & Lane == 'B')
+names(immdata$data)
+```
+
+### Method `by.repertoire` or `by.rep`
+
+#### Example 3
+
+Filter for repertoires  containing more than 60000 clonotypes:
+
+```{r example-3.1}
+#method = "by.repertoire", query = list(n_clonotypes = morethan(6000))
+#method = "by.rep", query = list(n_clonotypes = morethan(6000)) # Works both with by.rep and by.repertoire
+```
+
+Note, filtering `by.repertoire` or `by.rep` could change a number of repertoires or samples in your data: 
+
+```{r example-3.1.1}
+names(immdata$data)
+```
+
+Filter for repertoires containing less than 4000 clonotypes:
+
+```{r example-3.2}
+#method = "by.repertoire", query = list(n_clonotypes = lessthan(4000))
+#method = "by.rep", query = list(n_clonotypes = lessthan(4000)) # Works both with by.rep and by.repertoire
+names(immdata$data)
+```
+
+### Method `by.clonotype` or `by.cl`
+
+#### Example 4
+
+Filter for all noncoding clonotypes from `immdata` . As can be seen, `immdata` datasets doesn't contain any noncoding clonotypes:
+
+```{r example-4.1}
+#method = "by.clonotype", query = list(CDR3.aa = exclude("partial", "out_of_frame"))
+
+```
+
+Note, filtering `by.clonotype` or `by.cl` works within repertoire or sample. Repertoire or sample could be removed from your data only in case if all clonotypes in sample do not satisfy the condition: 
```

Comment:
```suggestion
Note, filtering `by.clonotype` or `by.cl` works within repertoire(sample). Repertoire(sample) could be removed from your data only in case if all clonotypes in sample do not satisfy the condition: 
```